src/main/java/io/oneko/docker/DockerRegistryPolling.java

```diff
@@ -67,23 +72,41 @@
 	@Scheduled(fixedDelay = 20000, initialDelay = 20000)
 	protected void checkDockerForNewImages() {
 		if (!this.pollingInProgress.getAndSet(true)) {
+			log.trace("Starting polling job");
```

Comment:
I think that if a log writes "Starting XYZ" then it should write "Finished XYZ" as well.
To keep things clear both log-statements should be written by the same method or at least twin-methods.

Reply:
Fixed this.